### ISSUE TITLE
Calcinator Causing Server Crash

### ISSUE THREAD

**Issue by Soundtoxin** [pahimar/Equivalent-Exchange-3#787]
Link to server crash report: http://sprunge.us/YYON
Link to fml-latest.log: http://sprunge.us/hNYP
Reasons to believe it's the Calcinator:
    >the entity crashing is Vegres, one of the players
    >Vegres posted in minecraft general the following
"So... uhh... apparently equivalent exchange's calcinator crashes the bradcraft server."

I may have to temporarily remove EE3 to stop the crash, but I'll likely add it back and continue filing bugs as soon as this is fixed.

**Comment by pahimar** [comment]
In the ee3 configuration change when it runs dynamic value assignment from Never to something else.

This has nothing at all to do with the Calcinator.

**Comment by pahimar** [comment]
Also please read closed issues and the crash log itself before assuming what the issue is. A players comments alone are not indicative of the source of a problem alone which I'd why at least I'm thankful you provided logs.

**Comment by Soundtoxin** [comment]
I assure you I wasn't assuming anything, I just considered the Calcinator bit as another clue in addition to the obviously-more-objective crashlogs. I'll admit my naming of the issue was poor, but I was in a rush to report it and get my server back up and be done with the whole thing. I had players waiting to get back on.

As for reading closed issues first, I'll consider it, but I'm fairly new to reporting issues in general, and not yet sure how much I want to get into this.

I provided the logs first, so I'm surprised you're making such a big deal out of quoting a player right at the end.

I'll change the dynamic value assignment in the config and wait on the next EE3 update to re-enable the mod. Thank you for your prompt replies.

**Comment by pahimar** [comment]
There is nothing you need to wait for - changing the configuration will fix the issue. Your choice though 

And what you see as "a big deal" is me commenting that your only troubleshooting appeared to be taking the players comments as the only cause or reason for the crash and reporting it without further investigation.

Also reading closed issues (or searching them) is like googling

**Comment by pahimar** [comment]
Damn phone

Is like googling around for solutions to a problem. You might find someone that had something similar and see the resolution. It's basic troubleshooting. Thus issue has been reported at least 3 separate times in the past month and the resolution is in those tickets which are at the top of the closed pile.

Not jumping down your throat, simply pointing out the tools out there that will more quickly get your server up and running in general when dealing with any mods.

**Comment by Soundtoxin** [comment]
Hey, Pahimar. I didn't bother checking the config until now, just wanted you to know that it wasn't set to never. Never isn't even the default. Do you have any other suggestions for a fix? Does this constitute my experience being different from the past issue you seem to compare it to?

**Comment by pahimar** [comment]
The issue you are seeing is that there are items stored in the energy-values.dat that no longer exist in the game (updates, mod removals) and that's why you are getting a netty (network code) null pointer exception because Minecraft's NBT serialization doesn't do effective null pointer protection.

I have it on my list of things to do to write my own serializer/deserializer for this process but it's been lower priority compared to other things. The solution is to remove the energy-value.dat file so that it regenerates or to use the config option to have it regenerate. Some time ago I changed the default from Never to When Mods Update.

So tldr, delete the energy-value.dat or get a newer version of EE3.

And the cause of this was explained in previous tickets, if you must know.

**Comment by Soundtoxin** [comment]
Thanks!
